jsonschema-core.xml

```diff
@@ -891,35 +891,32 @@
 {
     "title": "Feature list",
     "type": "array",
-        "items": [
-            {
-                "title": "Feature A",
-                "properties": {
-                    "enabled": {
-                        "$ref": "#/$defs/enabledToggle",
-                        "default": true
-                    }
+    "items": [
+        {
+            "title": "Feature A",
+            "properties": {
+                "enabled": {
+                    "$ref": "#/$defs/enabledToggle",
+                    "default": true
                 }
-            },
-            {
-                "title": "Feature B",
-                "properties": {
-                    "enabled": {
-                        "description": "If set to null, Feature B
-                                        inherits the enabled
-                                        value from Feature A",
-                        "$ref": "#/$defs/enabledToggle"
-                    }
+            }
+        },
+        {
+            "title": "Feature B",
+            "properties": {
+                "enabled": {
+                    "description": "If set to null, Feature B inherits the
+                                    enabled value from Feature A",
+                    "$ref": "#/$defs/enabledToggle"
                 }
             }
-        ]
-    },
+        }
+    ],
     "$defs": {
         "enabledToggle": {
             "title": "Enabled",
-            "description": "Whether the feature is enabled (true),
-                            disabled (false), or under
-                            automatic control (null)",
+            "description": "Whether the feature is enabled (true), disabled
```

Comment:
Please re-wrap this the way it was before.  The way this gets formatted in the plain text output this line ends up too wide.